client/src/pages/reports/Show.js

```diff
@@ -15,9 +15,9 @@ import LinkTo from "components/LinkTo"
 import Messages from "components/Messages"
 import {
   AnchorLink,
-  PageDispatchersPropType,
   jumpToTop,
   mapPageDispatchersToProps,
+  PageDispatchersPropType,
   useBoilerplate
 } from "components/Page"
```

Comment:
Please keep the order of the imports from a package alphabetically sorted (case-sensitive).